sky/cli.py

```diff
@@ -2575,13 +2575,13 @@ def spot_status(all: bool, refresh: bool):
         cache = spot_lib.load_job_table_cache()
         if cache is not None:
             readable_time = log_utils.readable_time_duration(cache[0])
-            job_table_json = (
+            job_table_payload = (
```

Comment:
nit, not important to fix: not sure why this was called `job_table_json` since this was never a json 😅 perhaps best to just call it `table_output` or something

Reply:
Ahh, good catch! Fixed. Thanks!